xml/System.Threading.Tasks/ValueTask.xml

```diff
@@ -847,7 +847,13 @@ Two <xref:System.Threading.Tasks.ValueTask> instances are equal when they wrap t
       <Docs>
         <summary>Gets a <see cref="T:System.Threading.Tasks.ValueTask" /> that may be used at any point in the future.</summary>
         <returns>The preserved <see cref="T:System.Threading.Tasks.ValueTask" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+## Remarks
+
+Because a <xref:System.Threading.Tasks.ValueTask> can be backed by a pooled <xref:System.Threading.Tasks.Sources.IValueTaskSource>, it is not safe to await an arbitrary <xref:System.Threading.Tasks.ValueTask> multiple times. You can use the <xref:System.Threading.Tasks.ValueTask.Preserve> method to convert this <xref:System.Threading.Tasks.ValueTask> into the instance backed by a regular <xref:System.Threading.Tasks.Task> that is safe to await multiple times. This method is similar to <xref:System.Threading.Tasks.ValueTask.AsTask>, but it returns the same <xref:System.Threading.Tasks.ValueTask> instance when this <xref:System.Threading.Tasks.ValueTask> represents a successful synchronously completed operation.
```

Comment:
```suggestion
Because a <xref:System.Threading.Tasks.ValueTask> can be backed by a pooled <xref:System.Threading.Tasks.Sources.IValueTaskSource>, it is not safe to await an arbitrary <xref:System.Threading.Tasks.ValueTask> multiple times. You can use the <xref:System.Threading.Tasks.ValueTask.Preserve> method to convert this <xref:System.Threading.Tasks.ValueTask> into the instance backed by a regular <xref:System.Threading.Tasks.Task> that is safe to await multiple times. This method is similar to <xref:System.Threading.Tasks.ValueTask.AsTask>, but it returns the same <xref:System.Threading.Tasks.ValueTask> instance when this <xref:System.Threading.Tasks.ValueTask> represents a successful synchronously completed operation. After calling <xref:System.Threading.Tasks.ValueTask.Preserve> the original <xref:System.Threading.Tasks.ValueTask> should never be used again
```